src/main/scala/com/codelouders/akkbbit/rabbit/RabbitMQService.scala

```diff
@@ -57,9 +65,13 @@ class RabbitMQService
   //do we need channel here?
   override def send(connection: RabbitMQConnection, data: ByteString): Boolean = {
     Try {
+      val queue = connection.connectionParams.queue.name
       val exchange = connection.connectionParams.exchange.map(_.name).getOrElse("")
```

Comment:
drop getOrElse here and match on Option below.

Reply:
Fix pushed :)